doc/hardware/peripherals/mipi.rst

```diff
@@ -0,0 +1,13 @@
+.. _mipi_api:
+
+MIPI
+####
+
+Overview
+********
+
+
+API Reference
+*************
+
+.. doxygengroup:: mipi_interface
```

Comment:
doxygen group should be called `mipi_dsi_interface`

Reply:
Done